molecule/shared/cleanup.yml

```diff
@@ -1,6 +1,5 @@
 - name: unregister systems
   hosts: all
-  gather_facts: false
```

Comment:
If the box it down then fact gathering will fail before the `wait_for_connection` task is run. Probably explicitly `- setup` before the unregistration task?

Reply:
[Outdated]

The newest version of the file handles that. F5 yourself. :+1:

Reply:
But I diiiiiiiiiiiiiiiiiid. /me punches a github. :D